### PR TITLE
chore(owners): configure pullapprove.com

### PR DESCRIPTION
**Pull request by alexeagle**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Configuration for pullapprove.com
+# See ownership spreadsheet:
+# https://docs.google.com/spreadsheets/d/1-HIlzfbPYGsPr9KuYMe6bLfc4LXzPjpoALqtYRYTZB0/edit?pli=1#gid=0&vpid=A5
```
Comment (vicb) [inline .pullapprove.yml]: `https://github.com/angular/angular/blob/master/TRIAGE_AND_LABELS.md#components` (public)

**Comment by vicb** [comment]
Could we override this (ie when someone is OOO / trivial changes) ?

**Comment by alexeagle** [comment]
We can always override it by merging even though some PR statuses are
non-green.

Note, this PR is stuck until we get pullapprove.com to fix their overly
broad OAuth permissions

**Comment by alexeagle** [comment]
I pushed the initial config as ed323402d1 since I enabled the pullapprove integration and it's making a funny looking message on new PRs.

**Pull request closed without merging**